can-value.md

```diff
@@ -17,6 +18,12 @@
     from(object, keyPath)
     // Returns an observable for only getting a property on an object.
 
+    fromValue(initialValue)
+    // Creates an observable with an initial value that can be read, written, and observed.
+
+    returnedBy(getter)
+    // Creates an observable that derives its value from other observable values.
+
```

Comment:
The main signature:
<img width="889" alt="screen shot 2018-07-02 at 11 01 51 pm" src="https://user-images.githubusercontent.com/10070176/42201278-f37d2fce-7e4b-11e8-9008-98e6199bd258.png">